astropy/coordinates/tests/test_representation.py

```diff
@@ -395,6 +442,27 @@ def test_getitem_scalar(self):
         with pytest.raises(TypeError):
             s_slc = s[0]
 
+    def test_representation_shortcuts(self):
+        """Test that shortcuts in ``represent_as`` don't fail."""
+        # diffs = UnitSphericalCosLatDifferential(4*u.mas/u.yr, 5*u.mas/u.yr,
```

Comment:
Sorry to be nitpicky, but could you add a comment here too? Nothing worse than commented-out code without a reason!
Fine to add a commit with `[ci skip]` in the commit message to avoid all the tests running again...